components/fxa-client/src/http_client.rs

```diff
@@ -343,6 +356,54 @@ impl FxAClient for Client {
         Ok(())
     }
 
+    fn attached_clients(
+        &self,
+        config: &Config,
+        refresh_token: &str,
+    ) -> Result<ResponseAndETag<Vec<GetAttachedClientResponse>>> {
+        let url = config.auth_url_path("v1/account/attached_clients")?;
+        let request =
+            Request::get(url).header(header_names::AUTHORIZATION, bearer_token(refresh_token))?;
+        let resp = Self::make_request(request)?;
+        let attached_clients: Vec<GetAttachedClientResponse> = resp.json()?;
+        let etag = resp
```

Comment:
Does this endpoint work with `etags`? I couldn't find anything in the FxA API docs.

Reply:
Yikes! I made an assumption without checking. Thanks for this catch.